index.js

```diff
@@ -242,6 +233,29 @@ function build (schema, options) {
   ajvInstance = new Ajv({ ...options.ajv, strictSchema: false, uriResolver: fastUri })
   ajvFormats(ajvInstance)
 
+  const validateDateTimeFormat = ajvFormats.get('date-time').validate
+  const validateDateFormat = ajvFormats.get('date').validate
+  const validateTimeFormat = ajvFormats.get('time').validate
+
+  ajvInstance.addKeyword({
```

Comment:
Why it introduces a keyword that is not part of JSON Schema here?

Reply:
I think we can optionally add the format from `ajv-format`. Then we add our own implementation of `date`, `date-time` and `time`.
So, it wouldn't need a non-JSON Schema keyword.

Reply:
You can add custom format only for string and number types.

Reply:
It does solve my question